database/seeders/EventsErstiwocheSeeder.php

```diff
@@ -329,11 +330,11 @@ public function runKaterbrunch(): void
         $event->name = 'Katerbrunch';
         $event->description = '<p>Nachdem wir alle nach der Kneipentour am Mittwoch Abend etwas verkatert sind, gibt es doch nichts besseres als zusammen bei einem guten Fr&uuml;hst&uuml;ck auszukatern 😊 <br />Hierf&uuml;r bitte wir euch die 2&euro; Anmeldegeb&uuml;hr am Montag zwischen 14:15 und 15:00 Uhr oder Mittwoch zwischen 09:30 und 13:00 Uhr im FSR zu bezahlen, sonst k&ouml;nnt ihr leider nicht teilnehmen.</p>
```

Comment:
The time slot where the money can be paied is not correct. Currently I am waiting for an answer for the correct time slot.

Reply:
13:45 - 16:00

Reply:
> 13:45 - 16:00

For monday and wednesday?

Reply:
> For monday and wednesday?

sorry, these are the current time slots:

Montag: 13:45 - 16:00
Mittwoch: 12:00 - 15:00